content/en/events/2023/kcseu/schedule.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Schedule
+type: docs
+weight: 3
+draft: true
+---
+
+**NOTE:** You must go through vaccine verification and pick up your badge before
+attending the summit or social evening event.
+
+## Steering AMA
+
+The Summit will start off with an AMA with members of the Kubernetes Steering
+Committee.  Bring any and all questions about how we're running the project,
+ongoing efforts & issues, and whatever you'd like to discuss with Steering.
+
+## Unconference
+
+The Contributor Summit will be an [Unconference-style event](https://blog.crisp.se/2016/08/30/henrikkniberg/what-is-an-unconference).  This means that
+you should arrive with topics in mind, and we will pitch them and vote on them
+in the morning, and then hold sessions in the afternoon, after the Steering AMA.
+This is the format used at many [DevOps Days](https://devopsdays.org/open-space-format/), and helps us accommodate changing 
+needs and unpredictable travel schedules.
+
+If you already have a topic in mind, we urge you to [add it to the Topics issue](https://github.com/kubernetes/community/issues/6633)
+so that other contributors can arrive mentally prepared for it.  This is
+especially crucial if you think you might not make it to the Summit until 
+afternoon.
+
+## SIG Meetings
+
+SIGs, Working Groups, and teams will be meeting during the Summit.  See if your
+team is meeting and mark your attendance on our 
+[SIG Meetings Schedule](https://docs.google.com/spreadsheets/d/1LS1q4VJajZkNZODdMnJ4eH3XnTsE_4em6pjQgToHyfw/edit?usp=sharing).
+
+## Doc Sprint
+
+SIG-Docs will have a full-day Documentation working session during the Summit.
+If you contribute to the docs -- and who doesn't -- or you want to learn to
+do it better, join them.
+
+## Schedule
+
+{{% schedule
+  text="Kubernetes Contributor Summit Europe 2023"
+  link="https://kcseu22.sched.com"
+  script="https://kcseu22.sched.com/js/embed.js"
```

Comment:
```suggestion
  text="Kubernetes Contributor Summit Europe 2023"
  link="https://kcseu23.sched.com"
  script="https://kcseu23.sched.com/js/embed.js"
```

Note: This will need to get provisioned first before it will render as an embed.

Reply:
Who does that typically? Is it a blocker to this being comitted?

Reply:
We should probably put "Schedule TBD" here instead, no?  Sched won't be available until March.